Bug: E2E tests failing - Cannot find module '@playwright/test'

## Description
E2E tests are not running. Error: Cannot find module '@playwright/test'

## Area
CI/CD / Frontend

## Priority
High - blocks test automation and CI pipeline

## Expected Behavior
E2E tests should run successfully in CI/CD pipeline

## Acceptance Criteria
- Make sure the e2e tests run on the fix branch on GitHub before merging
- Verify Playwright dependencies are properly installed
- E2E tests pass in CI pipeline

--- a/.github/workflows/e2e.yml
+++ b/.github/workflows/e2e.yml
@@ -38,10 +38,6 @@ jobs:
             apps/frontend/package-lock.json
             apps/backend/package-lock.json
 
-      - name: Install Playwright Browsers
-        working-directory: apps/frontend
-        run: npx playwright install --with-deps chromium
-
       - name: Install types package dependencies
         working-directory: packages/types
         run: npm ci
@@ -53,6 +49,10 @@ jobs:
       - name: Install frontend dependencies
         working-directory: apps/frontend
         run: npm ci
+
+      - name: Install Playwright Browsers
+        working-directory: apps/frontend
+        run: npx playwright install --with-deps chromium
 
       - name: Build frontend
         working-directory: apps/frontend

